FluidNC/src/Motors/Dynamixel2.cpp

```diff
@@ -73,7 +73,7 @@ namespace MotorDrivers {
 
         // servos will blink in axis order for reference
         LED_on(true);
-        vTaskDelay(100);
```

Comment:
this was probably a previous minor bug that should have been `vTaskDelay(100 / portTICK_PERIOD_MS);`